src/app/pages/HomePage/HomePage.tsx

```diff
@@ -107,7 +107,9 @@ const HomePage = ({ pageData }: HomePageProps) => {
                 return (
                   <React.Fragment key={`${curationId}-${position}`}>
                     <HomeCuration
-                      headingLevel={curationTitle ? 3 : 2}
+                      headingLevel={
```

Comment:
I think we can remove this now, because headingLevel has been removed from the Curation component above.

Reply:
done